### ISSUE TITLE
Customizer preview can't be displayed on WP 4.9.7

### ISSUE THREAD

**Issue by mintbird** [themeum/kirki#1944]
### Issue description:
After upgrading from WP 4.9.6 to WP 4.9.7, The customizer preview can't be displayed randomly. The problem is also occurred on other admin pages and front pages but not often like customizer page.

The problem was gone when downgrade to WP 4.9.6. I'm quite sure this issue is occurred only WP 4.9.7.

### Tested on browsers (updated to latest version before testing)

Chrome 67.0.3396.99 (Official Build) (64-bit)

![image](https://user-images.githubusercontent.com/695425/42428725-6b298cc6-835f-11e8-8a1b-8cf49803cdfb.png)

Opera 54.0.2952.51 Windows 10 64-bit (WoW64)

![image](https://user-images.githubusercontent.com/695425/42428736-73ce49d4-835f-11e8-85a6-ccbfb3518e98.png)

Microsoft Edge 42.17134.1.0

![image](https://user-images.githubusercontent.com/695425/42428730-6ffd0f20-835f-11e8-93b7-e9e2c216c4c3.png)

- I also tested on Firefox and iPad safari but seems those browser is auto reconnect after fails.
- There is no error logs (php+apache) that should be related to this issue.
- Sometime on front page i've seen php error about exceeding memory (Tried to allocate memory >1GB) but can't find a steps to reproduce this error.

### Version used:

- Clean installation of WP 4.9.7, No plugins activated.
- Twenty Seventeen theme (Version: 1.6)
- Kirki Dev (at [30 June 2018](https://github.com/aristath/kirki/tree/7a660bb590a8e8fba04af632b05842c5d9975969))
- Kirki 3.0.33 (this version also has the same issue)
- PHP 7.1.11 + PHP 7.1.18
- Apache 2.4.29
- Windows 10 

### Using theme_mods or options?
theme_mods

### Code to reproduce the issue (config + field(s))

1. Use clean installation for WP 4.9.7 and Twenty Seventeen theme (Version: 1.6)
2. Add the below code into `wp-content\themes\twentyseventeen\functions.php`
3. Go to 'Appearance > Customizer' and refresh several times (sometime the problem may not be occurred immediately).
4. Downgrade to WP 4.9.6 by using [this plugin](https://wordpress.org/plugins/wp-downgrade/) and check the customizer page again. The problem will be gone.

```
if ( class_exists( 'Kirki' ) ) {
	Kirki::add_panel( 'panel_general', array(
		'title'       => esc_html__( 'General', 'mono' ),
		'priority'    => 30,
	) );

	Kirki::add_section( 'section_general_typography', array(
		'title'          => esc_html__( 'Typography', 'mono' ),
		'panel'          => 'panel_general',
		'capability'     => 'edit_theme_options',
		'priority'       => 40,
	) );


	Kirki::add_field( 'mytheme', array(
		'type'        => 'dimension',
		'settings'    => 'vw_h2_font_size',
		'label'       => esc_html__( 'H2 Font Size', 'mono' ),
		'section'     => 'section_general_typography',
		'default'     => '30px',
		'output'      => array(
			array(
				'element' => 'h2',
				'property' => 'font-size',
			),
		),
		'transport'   => 'auto',
		'priority'    => 40,
	) );
}
```

**Comment by santenberger** [comment]
Hi everybody

First of all: Thank for this great plugin!

Same problem here. I was able to track the problem quite far. Here is what I found out:

It seems that "kirki::add_field" produces an error in Appache. When you disable the call to "kirki::add_field" the error_log in Appache stays clean. The error log is:

[Mon Jul 09 16:44:35.201751 2018] [ssl:warn] [pid 8604:tid 660] AH01909: www.example.com:443:0 server certificate does NOT include an ID which matches the server name
[Mon Jul 09 16:44:35.248623 2018] [mpm_winnt:notice] [pid 8604:tid 660] AH00455: Apache/2.4.25 (Win32) OpenSSL/1.0.2j PHP/7.0.20 configured -- resuming normal operations
[Mon Jul 09 16:44:35.248623 2018] [mpm_winnt:notice] [pid 8604:tid 660] AH00456: Apache Lounge VC14 Server built: Dec 17 2016 10:42:52
[Mon Jul 09 16:44:35.248623 2018] [core:notice] [pid 8604:tid 660] AH00094: Command line: 'c:\\xampp7\\apache\\bin\\httpd.exe -d C:/xampp7/apache'
[Mon Jul 09 16:44:35.279871 2018] [mpm_winnt:notice] [pid 8604:tid 660] AH00418: Parent: Created child process 4184
[Mon Jul 09 16:44:36.373664 2018] [ssl:warn] [pid 4184:tid 752] AH01909: www.example.com:443:0 server certificate does NOT include an ID which matches the server name
[Mon Jul 09 16:44:36.608023 2018] [ssl:warn] [pid 4184:tid 752] AH01909: www.example.com:443:0 server certificate does NOT include an ID which matches the server name
[Mon Jul 09 16:44:36.654893 2018] [mpm_winnt:notice] [pid 4184:tid 752] AH00354: Child: Starting 150 worker threads.
[Mon Jul 09 16:44:38.436292 2018] [mpm_winnt:notice] [pid 8604:tid 660] AH00428: Parent: child process 4184 exited with status 3221226356 -- Restarting.

This "Restarting" at the end happens (in my case...) about twenty times each call. The more fields that are activated the more often it restarts. It makes the customizer slow and sometimes it failes and comes with the message that there was no connection possible. But the result can't be foreseen. It's possible that you get a connection in one call and no connection in the next call. It's a little bit gambling :-)

I tracked the problem down to the file "kirki.php" around line 245.
Disabling the line "new $classname( $config_id, $args );" makes the Appache-error gone.

Hope that helps in finding the problem.

**Comment by lvasoul** [comment]
this is WP 4.9.7 bug, in 4.9.8 will be fixed at 31 july release
https://wordpress.org/support/topic/fatal-error-allowed-memory-size-of-11/

**Comment by niklasp** [comment]
can be closed possibly #2026

**Comment by aristath** [comment]
> can be closed possibly #2026 

Agreed, closed.
